oak_containers_orchestrator/src/key_provisioning.rs

```diff
@@ -13,17 +13,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::proto::oak::{
-    containers::{
-        orchestrator_key_provisioning_server::OrchestratorKeyProvisioning, SendGroupKeysRequest,
-    },
-    key_provisioning::v1::{
-        key_provisioning_server::KeyProvisioning, GetGroupKeysRequest, GetGroupKeysResponse,
+use crate::{
+    crypto::KeyStore,
+    proto::oak::{
+        containers::v1::{
+            orchestrator_key_provisioning_server::OrchestratorKeyProvisioning, SendGroupKeysRequest,
+        },
+        key_provisioning::v1::{
+            key_provisioning_server::KeyProvisioning, GetGroupKeysRequest, GetGroupKeysResponse,
+        },
     },
 };
+use std::sync::{Arc, OnceLock};
 use tonic::{Request, Response};
 
-struct KeyProvisioningService {}
+struct KeyProvisioningService {
+    _key_store: OnceLock<Arc<KeyStore>>,
```

Comment:
I would suggest waiting to add this until it is needed. I don't think this will require a OnceLock, but rather would likely require interior mutability of the group key inside the Key store.